src/Enable.Extensions.Queuing.AzureServiceBus/AzureServiceBusQueueClientFactory.cs

```diff
@@ -7,8 +7,14 @@ namespace Enable.Extensions.Queuing.AzureServiceBus
     public class AzureServiceBusQueueClientFactory : IQueueClientFactory
     {
         private readonly AzureServiceBusQueueClientFactoryOptions _options;
+        private readonly int _prefetchCount;
 
         public AzureServiceBusQueueClientFactory(AzureServiceBusQueueClientFactoryOptions options)
+            : this(options, 0)
+        {
+        }
+
+        public AzureServiceBusQueueClientFactory(AzureServiceBusQueueClientFactoryOptions options, int prefetchCount)
```

Comment:
Let's add a `PrefetchCount` property to `AzureServiceBusQueueClientFactoryOptions`. When doing this we will need to add validation to ensure that `PrefetchCount` is set to a valid value (for example, presumably we should not accept negative numbers, for an example, see https://github.com/Azure/azure-sdk-for-net/blob/4bab05144ce647cc9e704d46d3763de5f9681ee0/sdk/servicebus/Microsoft.Azure.ServiceBus/src/QueueClient.cs#L202-L205).